src/page.js

```diff
@@ -201,21 +201,25 @@ function appPage(creator, {storeSave, storeDelete, storeSet, refreshValues}) {
     );
   }
 
-  function onViewValue(itemId, element) {
+  function onViewValue(id, element) {
     analytics.fireEvent("view_value");
-    const textArea = document.getElementById(`token-${itemId}`);
+    const textArea = document.getElementById(`token-${id}`);
     textArea.disabled = true;
+    const keyDetails = document.getElementById(`key-${id}`);
+    keyDetails.classList.add("display-none");
```

Comment:
We want to show these values when clicking `view value`
```suggestion
    keyDetails.classList.remove("display-none");
```

Reply:
Removed ✅